packages/bbui/src/Form/Multiselect.svelte

```diff
@@ -1,40 +1,41 @@
-<script>
+<script lang="ts">
   import { createEventDispatcher } from "svelte"
   import Multiselect from "./Core/Multiselect.svelte"
   import Field from "./Field.svelte"
 
-  export let value = []
-  export let label = null
+  export let value: string[] | string = []
+  export let label: string | undefined = undefined
   export let disabled = false
   export let readonly = false
   export let labelPosition = "above"
-  export let error = null
-  export let placeholder = null
-  export let options = []
-  export let getOptionLabel = option => option
-  export let getOptionValue = option => option
+  export let error: string | undefined = undefined
+  export let placeholder: string | undefined = undefined
+  export let options: any[] = []
```

Comment:
We should type these any from generics, so the options in function will be typed. 
```
<script lang="ts" generics="O">
export let options: O[] = []
export let getOptionLabel = (option: O) => option
```